src/atoms/google-analytics/google-analytics.tsx

```diff
@@ -24,5 +25,59 @@ const GoogleAnalyticsBase = ({ config }) => {
   return <Analytics type="googleanalytics" targets={gaConfig} />;
 };
 
+const GoogleAnalytics4 = ({config}) => {
+  const Ga4ConfigObject = get(config, ["opts", "featureConfig", "ga4Config"], null);
+  return <Analytics type="googleanalytics" targets={Ga4ConfigObject} config="./ga4.json" data-credentials="include"/>;
```

Comment:
`config="./ga4.json"` this will not work. How will it get the config.json? where is it hosted please check